tests/components/reolink/conftest.py

```diff
@@ -137,6 +137,8 @@ def reolink_connect_class() -> Generator[MagicMock]:
         host_mock.daynight_state.return_value = "Black&White"
         host_mock.hub_alarm_tone_id.return_value = 1
         host_mock.hub_visitor_tone_id.return_value = 1
+        host_mock.recording_packing_time_list = ["30 Minutes", "60 Minutes"]
```

Comment:
How does the API return this?

Reply:
The API gives a string "30 Minutes", to change you also need to send "30 Minutes".

I think it would have been cleaner if the API would not have send the "Minutes", but it is what it is.
I thought about stripping the "Minutes" in the upstream library, but that would just be more code.